### PR TITLE
Change db table and 'new listing' form

### PR DESCRIPTION
**Pull request by raptorf1**
# [PT Story](https://www.pivotaltracker.com/story/show/165866036)

# User Story
```
As a development team
In order to align database and form to the new structure
We want to change the labels of the DB columns and form fields to match the new structure
```

# Changes we propose
* Edit the columns of the Listing database, since we changed the direction of the application on Tuesday.
* Edit the fields of the 'new listing' form, since the database columns are changed.
* Edit all relevant feature and unit tests to reflect the above mentioned changes.

# Things we have learned
* Adding columns to a database
* Removing columns to a database
* Touched a tiny bit upon Active Record

### CODE REVIEW

File: db/migrate/20190508150729_add_start_date_end_date_pet_picture_url_to_listings.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddStartDateEndDatePetPictureUrlToListings < ActiveRecord::Migration[5.2]
+  def change
+    add_column :listings, :start_date, :date
+    add_column :listings, :end_date, :date
+    add_column :listings, :pet_picture, :string
+  end
+end
```
Comment (leiter007) [inline db/migrate/20190508150729_add_start_date_end_date_pet_picture_url_to_listings.rb:7]: Is there a reason why this became four migration files (2x removing and 2x adding)? Maybe you tested with one file first to see it worked or?

**Review by leiter007 — COMMENTED** [review]
Added super small comment. Reviewed and looks good, DB changes seem to be as we discussed!

File: db/migrate/20190508150729_add_start_date_end_date_pet_picture_url_to_listings.rb (hunk 1 above)
Comment (raptorf1) [reply db/migrate/20190508150729_add_start_date_end_date_pet_picture_url_to_listings.rb:7]: it's a learning experience man :P 
we tested with Remove...From + Add...To

**Review by boamatule — COMMENTED** [review]
Well structured and all def well done. Good job guys.

File: db/migrate/20190508150729_add_start_date_end_date_pet_picture_url_to_listings.rb (hunk 1 above)
Comment (leiter007) [reply db/migrate/20190508150729_add_start_date_end_date_pet_picture_url_to_listings.rb:7]: Always great stuff with great learning experiences!!!

**Pull request merged**